cypress/private/cypress_test.bzl

```diff
@@ -9,31 +9,25 @@ _attrs = dicts.add(js_binary_lib.attrs, {
     ),
 })
 
-# Do the opposite of _to_manifest_path in
-# https://github.com/bazelbuild/rules_nodejs/blob/8b5d27400db51e7027fe95ae413eeabea4856f8e/nodejs/toolchain.bzl#L50
-# to get back to the short_path.
-# TODO: fix toolchain so we don't have to do this
-def _target_tool_short_path(path):
-    return ("../" + path[len("external/"):]) if path.startswith("external/") else path
-
 def _impl(ctx):
-    cypress_bin = _target_tool_short_path(ctx.toolchains["@aspect_rules_cypress//cypress:toolchain_type"].cypressinfo.target_tool_path)
+    cypress_bin = ctx.toolchains["@aspect_rules_cypress//cypress:toolchain_type"].cypressinfo.target_tool_path
     cypress_files = ctx.toolchains["@aspect_rules_cypress//cypress:toolchain_type"].cypressinfo.tool_files
 
     files = ctx.files.data[:] + cypress_files + ctx.files.browsers
 
-    if ctx.attr.chdir:
-        cypress_bin = "/".join([".." for _ in ctx.attr.chdir.split("/")] + [cypress_bin])
+    # TODO: Augment rules_js to expose EXECROOT before setting fixed_env. Then use that variable.
+    entry_point_dir = '"$$( dirname "$$( realpath "$${BASH_SOURCE[0]}" )" )"'
+    entry_point_offset = "../../../.."
+    execroot = "/".join([entry_point_dir, entry_point_offset] + [".." for _ in ctx.attr.name.split("/")])
+    cypress_run_binary = "/".join([execroot, cypress_bin])
 
     launcher = js_binary_lib.create_launcher(
         ctx,
         log_prefix_rule_set = "aspect_rules_cypess",
         log_prefix_rule = "cypress_node_test",
         fixed_args = ctx.attr.fixed_args,
         fixed_env = {
-            "CYPRESS_RUN_BINARY": cypress_bin,
-            "HOME": "$$TEST_TMPDIR",
```

Comment:
`TEST_TMPDIR` isn't defined in `bazel run` so I removed these. I'd like to keep them and use something like `or` if they aren't defined. Any suggestions @gregmagolan ?

Reply:
Why did we set `HOME` in the first place? Guessing cypress was trying to write to the home directory when it the sandbox and failing?